2FA & access pruning (#440)
ropensci/dev_guide: pkg_security.Rmd

This work-in-progress chapter includes [guidance about managing secrets in packages](#pkgsecrets) and [links for further reading](#furthersecreading).
```

> ## Misc

Reviewer: Just wondering why "Misc" and not the actual word "Miscellaneous"?

Author: Oops, changed! For the record the diff was first to remove the section then to re-add it.

We recommend the article [Ten quick tips for staying safe online](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008563) by Danielle Smalls and Greg Wilson.

## GitHub access security

* We recommend you [secure your GitHub account with two-factor (authentication) 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/). It is *compulsory* for all ropensci GitHub organization members and outside collaborators so make sure to enable it before your package is approved.

> * We also recommend you regularly check who has access to your package repository, and that you prune any useless access (e.g. of a former contractor).

Reviewer: ```suggestion
* We also recommend you regularly check who has access to your package repository, and that you prune any unused access (such as from former collaborators).
```
